test/test_time_series.jl

```diff
@@ -211,6 +211,36 @@ end
     forecast_retrieved =
         IS.get_time_series(IS.Probabilistic, component, "test"; start_time = initial_time)
     @test IS.get_initial_timestamp(forecast_retrieved) == initial_time
+    t = IS.get_time_series_array(
+        IS.Probabilistic,
+        component,
+        "test";
+        start_time = initial_time,
+    )
+    @test size(t) == (24, 99)
+end
+
+@testset "Test add Scenarios" begin
+    initial_time = Dates.DateTime("2020-09-01")
+    resolution = Dates.Hour(1)
+    other_time = initial_time + resolution
+    name = "test"
+    horizon = 24
+    data_vec = Dict(initial_time => ones(horizon, 99), other_time => ones(horizon, 99))
+    d = data_vec
+    sys = IS.SystemData()
+    component_name = "Component1"
+    component = IS.TestComponent(component_name, 5)
+    IS.add_component!(sys, component)
+    forecast = IS.Scenarios(name, d, resolution)
+    IS.add_time_series!(sys, component, forecast)
+    @test IS.has_time_series(component)
+    @test IS.get_initial_timestamp(forecast) == initial_time
+    forecast_retrieved =
+        IS.get_time_series(IS.Scenarios, component, "test"; start_time = initial_time)
+    @test IS.get_initial_timestamp(forecast_retrieved) == initial_time
+    t = IS.get_time_series_array(IS.Scenarios, component, "test"; start_time = initial_time)
+    @test size(t) == (24, 99)
```

Comment:
- The test should use unique data and then verify it after the read.
- The test should include permutations that vary the start_time, length, and count. Consider whether we need something similar to the testset labeled `"Test get_time_series_array Deterministic"`.